android/build.gradle

```diff
@@ -188,6 +193,21 @@ android {
             }
         }
     }
+    tasks.withType(ExternalNativeBuildJsonTask) {
+        compileTask ->
+            compileTask.doLast {
+                if (!isGHExampleApp()) {
+                    return
+                }
+
+                def rootDir = new File("${project.projectDir}/..")
+                def generated = new File("${compileTask.abi.getCxxBuildFolder()}/compile_commands.json")
+                def output = new File("${rootDir}/compile_commands.json")
+                output.text = generated.text
+
+                println("Generated clangd metadata.")
+            }
+    }
```

Comment:
I think this entire thing should be executed conditionally instead of early return inside.

Reply:
Still up to date